mantidimaging/gui/dialogs/cor_inspection/view.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import, division, print_function
+
+from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg
+from matplotlib.figure import Figure
+
+from mantidimaging.gui.mvp_base import BaseDialogView
+from mantidimaging.gui.utility import BlockQtSignals
+from mantidimaging.gui.widgets import NavigationToolbarSimple
+
+from .presenter import (
+        CORInspectionDialogPresenter,
+        Notification as PresNotification)
+
+
+class CORInspectionDialogView(BaseDialogView):
+    def __init__(self, parent, cmap='Greys_r', **args):
+        super(CORInspectionDialogView, self).__init__(
+                parent, 'gui/ui/cor_inspection_dialog.ui')
+
+        self.presenter = CORInspectionDialogPresenter(self, **args)
+        self.cmap = cmap
+
+        # Completed button action
+        self.finishButton.clicked.connect(self.accept)
+
+        # Image canvas
+        self.image_figure = Figure(tight_layout=True)
+        self.image_canvas = FigureCanvasQTAgg(self.image_figure)
+        self.image_canvas.setParent(self)
+        self.imagePlotLayout.addWidget(self.image_canvas)
+        self.image_plots = self.image_figure.subplots(
```

Comment:
I had `matplotlib` `2.02` installed and that doesn't have `Figure.subplots` - it looks like that was introduced in `2.1.0`. Should we have a minimum `matplotlib` version requirement?

Reply:
Done.